zipkin-ui/js/component_ui/traceToMustache.js

```diff
@@ -106,8 +106,12 @@ export default function traceToMustache(trace) {
       const spanStartTs = span.timestamp || traceTimestamp;
       const spanDepth = spanDepths[span.id] || 1;
       const width = (span.duration || 0) / summary.duration * 100;
+      let isError = false;
```

Comment:
not sure, but I *think* this function is unit tested, if accessible, mind adding a test for 'error' -> isError?